string action
Waypoint[] waypoints
---
bool ack

